### PR TITLE
Version 3.0.0-pre.1389 - October 23, 2023

### PR DESCRIPTION
**Pull request by tejal-shah**
### Features/Improvements
- Dot plots can display **standard deviation** and **mean absolute deviation** [#181914408](https://www.pivotaltracker.com/story/show/181914408)
- Close component should be undoable as a single action [#186075314](https://www.pivotaltracker.com/story/show/186075314)
- A newly created map with no data to display defaults to user location [#186252775](https://www.pivotaltracker.com/story/show/186252775)
- Update all adornment model type properties that are currently set to simple strings [#186297171](https://www.pivotaltracker.com/story/show/186297171)

### Bug Fixes
- Wrong attribute sometimes passed to formula in **Plotted Value** [#186266057](https://www.pivotaltracker.com/story/show/186266057)
- Evaluate plotted value adornment formula [#186267057](https://www.pivotaltracker.com/story/show/186267057)
- Unable to add attribute to graph in presence of two datasets [#186316077](https://www.pivotaltracker.com/story/show/186316077)

### CODE REVIEW

File: v3/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## Version 3.0.0-pre.1389 - October 24, 2023
+
+### Features/Improvements
+- Close component should be undoable as a single action [#186075314](https://www.pivotaltracker.com/story/show/186075314)
+
+### Bug Fixes
+- Wrong attribute sometimes passed to formula in **Plotted Value** [#186266057](https://www.pivotaltracker.com/story/show/186266057)
```
Comment (kswenson) [inline v3/CHANGELOG.md:9]: Seems like we may be missing some features/improvements/bug fixes. Just looking at the recently merged PRs: #946, #949, #951, #952, #953, #956 could all warrant a mention.

**Review by kswenson — CHANGES_REQUESTED** [review]
👍 Looks good, but I think there are some more improvements that could be mentioned.

File: v3/CHANGELOG.md (hunk 1 above)
Comment (tejal-shah) [reply v3/CHANGELOG.md]: Thank you for catching that @kswenson . I've added these now (#949 was already there; I've the remaining 5 and compared the git log output against the changelog list to make sure I haven't missed anything else).

**Review by kswenson — APPROVED** [review]
👍 LGTM

**Pull request merged**